handlers/restart_consul.yml

```
@@ -6,13 +6,13 @@
   when: ansible_service_mgr == "systemd"
   listen: 'reload systemd daemon'
 
-- name: Restart consul on Unix
-  service:
-    name: consul
-    state: restarted
-    # Needed to force SysV service manager on Docker for Molecule tests
-    use: "{{ ansible_service_mgr }}"
+- name: Rolling restart consul on Unix
+  # Manual loop on playbook hosts to perform a "serial: 1" equivalent on a set of task
+  include_tasks: ../tasks/leave_restart_consul.yml
+  with_items: "{{ play_hosts }}"
```

It seems slightly dangerous to me to use Ansible's default `item`. Namely, if someone is invoking this role from inside of a loop, then we'll end up shadowing that variable.

```suggestion
  with_items: "{{ play_hosts }}"
  loop_control:
    loop_var: rolling_restart_host
```